agent/metadata_service.go

```diff
@@ -132,7 +130,7 @@ func (mds *metadataService) getCredentials(w http.ResponseWriter, r *http.Reques
 		Code:            "Success",
 		LastUpdated:     time.Now().UTC().Format(time.RFC3339),
 		Type:            "AWS-HMAC",
-		AccessKeyId:     creds.AccessKeyId,
+		AccessKeyID:     creds.AccessKeyId,
```

Comment:
This worries me, as I'm not sure how AWS SDK programs will react to the potential change in the "D" capitalization there. Have you tried this build running on a laptop yet?